spec/suites/map/MapSpec.js

```diff
@@ -611,6 +611,62 @@ describe("Map", function () {
 		});
 	});
 
+	describe("#addHandler", function () {
+		function getHandler(callback = () => {}) {
+			return L.Handler.extend({
+				addHooks: function () {
+					L.DomEvent.on(window, 'click', this.handleClick, this);
+				},
+
+				removeHooks: function () {
+					L.DomEvent.off(window, 'click', this.handleClick, this);
+				},
+
+				handleClick: callback
+			});
+		}
+
+		it("checking enabled method", function () {
+			L.ClickHandler = getHandler();
+			map.addHandler('clickHandler', L.ClickHandler);
+
+			map.clickHandler.enable();
```

Comment:
Let us add a check, if it's disabled by default
```suggestion
			map.addHandler('clickHandler', L.ClickHandler);

			expect(map.clickHandler.enabled()).to.eql(false);

			map.clickHandler.enable();
```